src/main/webapp/assets/modules/common/vnwConfigService.js

```diff
@@ -0,0 +1,737 @@
+techlooper.factory("vnwConfigService", function (jsonValue, $translate, $rootScope, utils) {
```

Comment:
Line is too long.<br>'techlooper' is not defined.<br>'utils' is defined but never used.<br>'$rootScope' is defined but never used.